addons/sale/models/sale.py

```diff
@@ -36,6 +36,9 @@ def _get_default_require_signature(self):
     def _get_default_require_payment(self):
         return self.env.user.company_id.portal_confirmation_pay
 
+    def _get_default_required_payment_percentage(self):
```

Comment:
guidelines: `_default_<field_name>`

Reply:
Do i also change the two other default methods (for require signature and require payment) while i'm on it?

Reply:
No, no useless change. Moreover, it's not the purpose of the branch ;)